crates/bitwarden-cli/src/lib.rs

```diff
@@ -15,3 +15,29 @@ pub fn text_prompt_when_none(prompt: &str, val: Option<String>) -> InquireResult
         Text::new(prompt).prompt()?
     })
 }
+
+/// Try to get a value from CLI arg, then from environment variables, then prompt
+///
+/// Checks multiple environment variable names in order (e.g., BW_CLIENTID, BW_CLIENT_ID)
+pub fn text_or_env_prompt(
```

Comment:
```suggestion
pub fn resolve_user_input_value(
```

Just a suggestion. This name seems a little vague.

Reply:
thank you for this feedback! I've replaced the function name and usages in my last commit.

Also wondering about renaming `text_prompt_when_none` for a more proper name, don't hesitate to point out your suggestion please:

https://github.com/bitwarden-labs/sdk-internal/blob/9236ae5a0b98453598926aeed9a175588596732d/crates/bw/src/auth/login.rs#L1